source/index.html.md

```diff
@@ -15,45 +16,37 @@ code_clipboard: true
 ---
 # Introduction
 
-Betable API is a RESTful service providing the ability to create a compliant wallet solution while powering CRM, fraud, and payment facilities for a regulated gaming solution.
+The Betable API is a RESTful service providing the ability to create a compliant gaming solution with integrated wallet, CRM, fraud, and payment facilities.
 
 ## Change History
 
 Version | Date | Summary of Changes
 --------- | ------- | -----------
-1.0 | 2020-03-23 | Initial release of Betable-LFS API.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+1.0 | 2020-03-23 | Initial release of Betable API.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 
 ## Quick Reference
 
-All API access is over HTTPS, and accessed from the `ns-api.betable.com` domain. All API requests must be authenticated or a `401 Unauthorized error` will be returned (see Authentication).
+All API access is over HTTPS and must be authenticated or a `401 Unauthorized error` will be returned (see Authentication).
 
-A `User` corresponds to each unique player in the vendor system and contains profile and preference information.
+A `User` corresponds to each unique player in the partner system and contains profile and preference information.
```

Comment:
II did a find/replace on all `vendor` with `partner` as they are not a vendor (to us) as we are not buying something from them, we actually are the vendor to them.